website/docs/content/punctuation/punctuation.md

```diff
@@ -4,11 +4,196 @@ title: Punctuation and special symbols
 
 ## Dashes and hyphens
 
-### Em dash (—)
+### Em dash
 
 | Symbol       | HTML entity        |
 | :----------: | :----------------: |
 | &mdash;      | `&mdash;`          |
 
 * Use sparingly in sentences in place of a comma or colon
 * Omit the spaces on both sides of the dash
+
+![](/content/punctuation/static/emdash.jpg)
```

Comment:
Nastya, can you please use png format for illustrations? Png saves image quality better. You also can compress them using the [TinyCanvas Figma plugin](https://www.figma.com/community/plugin/794829197801903069/tinycanvas) when exporting from Figma.